src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/IProjectHotReloadAgent.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.HotReload
+{
+    public interface IProjectHotReloadAgent
+    {
+        IProjectHotReloadSession? CreateHotReloadSession(string runtimeVersion, IProjectHotReloadSessionCallback callback);
```

Comment:
@BillHiebert My intention here is that `CreateHotReloadSession` will return `null` if it can't create the session for whatever reason. E.g., it can't find a a delta applier that matches `runtimeVersion`.

Reply:
@tmeschter that seems reasonable.